dbms/src/Storages/CMakeLists.txt

```diff
@@ -17,6 +17,10 @@ add_subdirectory (Page)
 add_subdirectory (DeltaMerge/File/dtpb)
 add_subdirectory (DeltaMerge/tools)
 
+if (ENABLE_V3_PAGESTORAGE)
```

Comment:
Remove `ENABLE_V3_PAGESTORAGE` in all source files.
cc @JaySon-Huang

Reply:
fixed!